tests/Unit/Services/VoucherEvaluator/VoucherEvaluatorTest.php

```diff
@@ -170,74 +264,34 @@ public function itCreditsWhenAChildIsUnderSchoolAge()
         $this->assertEquals(1, count($credits));
 
         // Check the correct credit type is applied.
-        $this->assertNotContains(self::CREDIT_TYPES['ChildIsUnderOne'], $credits, '');
-        $this->assertContains(self::CREDIT_TYPES['ChildIsUnderSchoolAge'], $credits, '');
+        $this->assertContains(self::CREDIT_TYPES['ChildIsBetweenOneAndPrimarySchoolAge'], $credits, '');
```

Comment:
What's the empty string for?

Reply:
normally, a message to return; redundant, will remove